scripts/remove/lounge.sh

```diff
@@ -5,8 +5,13 @@ systemctl stop -q lounge
 
 npm uninstall -g thelounge --save >> /dev/null 2>&1
 
-deluser lounge >> /dev/null 2>&1
-rm -rf /home/lounge
+deluser lounge --remove-home >> /dev/null 2>&1
+rm -rf /home/lounge # just in case
+
+if [[ -f /install/.nginx.lock ]]; then
+    rm /etc/nginx/apps/lounge.conf
```

Comment:
There's no need for the `if` block here. If you use `rm -f` it will remove the file without any warning if it does not exist.

We should be removing this file if it exists regardless of whether or not nginx is installed.

Reply:
I thought people who did subdomains basically installed nginx, removed the lock and played with the configs that way. But can do.

Reply:
I'm not aware of anyone doing this 🤔 

That said it should be `rm -f` to avoid the warning that the file does not exist -- `-f` ignores printing does not exist warnings to stderr.

Reply:
Ah, yeah. Done!